src/test/ArrayVariableGenerator.test.ts

```diff
@@ -0,0 +1,21 @@
+import ArrayVariableGenerator from '../models/Generators/ArrayVariableGenerator';
+import { ArrayVariableGeneratorParams } from '../models/Generators/ArrayVariableGenerator';
+
+test("test compute function is defined", () => {
```

Comment:
Este test esta bien pero creo que podria cambiar la descripcion, ahi estas diciendo que estas probando si *la funcion* esta definida, no el valor de retorno
podria ser mejor algo como
```suggestion
test("test compute function return value is defined if given a non-empty array", () => {
```